pypeclub/OpenPype
# Maya: add loaded containers to published instance

## Enhancement

Adding ability to include loaded containers to published instance.

This is mainly useful for Layouts and Setdress families. When you create them from loaded models for example, this will pull their containers along published Layout so OpenPype will retain information about them when Layout is loaded into the new scene.

This can be set in the Settings (`project_settings/maya/publish/ExtractMayaSceneRaw/add_for_families`):

![image](https://user-images.githubusercontent.com/33513211/156603778-24b81c34-585e-4a72-b7ad-2e0c95f360d0.png)

### How to test:

In the new scene load bunch of models with OpenPype, create Layout and publish it. When you load this layout, you should see all content in the layout in Manager.

Close #2830
